apps/api/src/app/events/usecases/send-message/digest/digest.usecase.ts

```diff
@@ -6,17 +6,19 @@ import { SendMessageType } from '../send-message-type.usecase';
 import { StepTypeEnum, DigestTypeEnum } from '@novu/shared';
 import { GetDigestEventsRegular } from './get-digest-events-regular.usecase';
 import { GetDigestEventsBackoff } from './get-digest-events-backoff.usecase';
+import { CreateExecutionDetails } from '../../../../execution-details/usecases/create-execution-details/create-execution-details.usecase';
 
 @Injectable()
 export class Digest extends SendMessageType {
   constructor(
     protected messageRepository: MessageRepository,
     protected createLogUsecase: CreateLog,
+    protected createExecutionDetails: CreateExecutionDetails,
     protected jobRepository: JobRepository,
     private getDigestEventsRegular: GetDigestEventsRegular,
     private getDigestEventsBackoff: GetDigestEventsBackoff
   ) {
-    super(messageRepository, createLogUsecase);
+    super(messageRepository, createLogUsecase, createExecutionDetails);
```

Comment:
What's the typical naming pattern we use for use cases? Adding `UseCase` or not?

Reply:
We do not have a pattern, my pattern is that I add it if something else already use that name of the usecase...

Reply:
I was mostly adding the `usecase` part only on the local usage, and the class itself won't have the usecase.

For example:

```
constructor(private createExecutionDetailsUsecase: CreateExecutionDetails)
```

But I'm open to standardized this in any other way that seems fit. Will add this to discuss on next Tech Summit